### DISCUSSION TITLE
Consider Modelio UML designer

### DISCUSSION THREAD

**Discussion by iamjuing** [StateSmith/StateSmith#140] [Ideas]
Although I like drawio and its support by StateSmith, it is still not a convenient tool for UML modeling.
Have you considered to support open source [Modelio](https://github.com/ModelioOpenSource/Modelio) which targets to UML.

**Comment by adamfk** [comment]
I have not looked at Modelio seriously. Do you use it?

A few years ago I tried some other eclipse based modelling tools and they weren't flexible enough for my needs.
